<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialInterfaceDialog</class>
 <widget class="QDialog" name="SerialInterfaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备接口</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>串口号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>COMA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COMB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COMC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COMD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COME</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COMF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COMG</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>波特率 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>230400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>384000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>460800</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>使能发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_6">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>禁止发送</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>使能发送</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_7">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>1位停止位</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2位停止位</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>奇偶校验</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>偶校验</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>奇校验</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>使能校验</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_8">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>禁止</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>使能奇偶校验</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>收发</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>发送</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>接收</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_8" native="true">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>协议</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>发送脉冲数</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>发送导航数据</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>装订</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
